src/main/java/net/masterthought/cucumber/Configuration.java

```diff
@@ -136,13 +138,31 @@ public String getProjectName() {
         return projectName;
     }
 
+    /**
+     * Sets directory qualifier.
+     *
+     * @param directoryQualifier directory qualifier
+     */
+    public void setDirectoryQualifier(String directoryQualifier) {
+        this.directoryQualifier = directoryQualifier;
+    }
+
+    /**
+     * Returns directory qualifier
+     *
+     * @return directory qualifier
+     */
+    public String getDirectoryQualifier() {
+        return StringUtils.defaultString(directoryQualifier);
+    }
+
     /**
      * Gets directory where the attachments are stored.
      *
      * @return directory for attachment
      */
     public File getEmbeddingDirectory() {
-        return new File(getReportDirectory().getAbsolutePath(), ReportBuilder.BASE_DIRECTORY
+        return new File(getReportDirectory().getAbsolutePath(), ReportBuilder.BASE_DIRECTORY + this.getDirectoryQualifier()
```

Comment:
looks like directory separator is missing here

Reply:
No, it's not missing; it's not needed - as this is just a suffix to the current directory name.  It can be passed as `_abcd` for example.